LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2019, Action is Hope
```

Comment:
Which license is it? Could be helpful for people to have the name of it somewhere